mantle/kola/harness.go

```diff
@@ -448,13 +448,16 @@ func ParseDenyListYaml(pltfrm string) error {
 		}
 
 		if obj.SnoozeDate != "" {
-			snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
+			snoozeDate, err := time.Parse(SnoozeFormat, obj.SnoozeDate)
 			if err != nil {
 				return err
 			} else if today.After(snoozeDate) {
+				// if snoozeDate within last 7 days, print aditional info
+				if snoozeDate.Before(snoozeDate.AddDate(0, 0, 7)) {
+					fmt.Printf("⏰ Snooze expired \"%s\":\n", snoozeDate.Format("Jan 02 2006"))
```

Comment:
this part should now be handled by https://github.com/coreos/coreos-assembler/pull/3539 - sorry I didn't realize the overlap.